user/views.py

```diff
@@ -1,3 +1,444 @@
+        except Exception as ex:
+            return JsonResponse({'message' : 'ERROR_' + ex.args[0]}, status = 400)
+
+class CheckID(View): # 아이디 중복확인
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            if data['user_id'] == '':
+                return JsonResponse({'message' : 'NOT_ENTERED_USER_ID'}, status = 400)
+
+            if User.objects.filter(user_id = data['user_id']).exists():
+                return JsonResponse({'message' : 'USER_ID_DUPLICATED'}, status = 400)
+            else:
+                return JsonResponse({'message' : 'SUCCESS'}, status = 200)
+
+        except KeyError as ex:
+            return JsonResponse({'message' : 'KEY_ERROR_' + ex.args[0]}, status = 400)
+        except Exception as ex:
+            return JsonResponse({'message' : 'ERROR_' + ex.args[0]}, status = 400)
+
+class CheckEmail(View): # 이메일 중복확인
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            if data['email'] == '':
+                return JsonResponse({'message' : 'NOT_ENTERED_EMAIL'}, status = 400)
+
+            p = re.compile('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
+            if p.match(str(data['email'])) == None:
+                return JsonResponse({'message':'EMAIL_VALIDATION'}, status = 400)
+
+            if User.objects.filter(email = data['email']).exists():
+                return JsonResponse({'message' : 'EMAIL_DUPLICATED'}, status = 400)
+            else:
+                return JsonResponse({'message' : 'SUCCESS'}, status = 200)
+
+        except KeyError as ex:
+            return JsonResponse({'message' : 'KEY_ERROR_' + ex.args[0]}, status = 400)
+        except Exception as ex:
+            return JsonResponse({'message' : 'ERROR_' + ex.args[0]}, status = 400)
+
+class SignIn(View): # 로그인
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            if User.objects.filter(user_id = data['user_id']).exists():
+                user_data = User.objects.get(user_id = data['user_id'])
+
+                if bcrypt.checkpw(data['password'].encode('utf-8'), user_data.password.encode('utf-8')) == False:
+                    return JsonResponse({'message' : 'INVALID_USER'}, status = 400)
+                
+                access_token = jwt.encode({'user_id' : data['user_id']}, SECRET, algorithm = ALGORITHM)
+
+                user_dic = {}
+                user_dic['id']            = user_data.id
+                user_dic['user_name']     = user_data.user_name
+                user_dic['email']         = user_data.email
+                user_dic['phone']         = user_data.phone
+                user_dic['address']       = user_data.address
+                user_dic['date_of_birth'] = user_data.date_of_birth
+                user_dic['user_rank']     = UserRank.objects.filter(user = user_data.id).get().name
```

Comment:
```suggestion
                user_dic = {
                    'id' : user_data.id,
                    'user_name' : user_data.user_name....
                }             
```
과 같이 딕셔너리 선언과 함께 바로 키: 밸류를 넣어주시는 것이 조금 더 깔끔합니다. 빈 딕셔너리를 선언하지 않아도 되고, 딕셔너리의 이름을 사용하지 않아도 되기 때문입니다 :)

Reply:
수정했습니다. 이렇게 하니까 코드가 훨씬 깔끔해진거같아요